### ISSUE TITLE
Event names are barely legible

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-qt-browser#130]
MWE:
```python
# %%
import mne

sample_dir = mne.datasets.sample.data_path()
sample_fname = sample_dir / 'MEG' / 'sample' / 'sample_audvis_raw.fif'

raw = mne.io.read_raw_fif(sample_fname)
raw.crop(tmax=60)

events = mne.find_events(raw, stim_channel='STI 014')
event_id = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
            'visual/right': 4, 'face': 5, 'buttonpress': 32}


raw.plot(events=events, event_id=event_id)
```
Produces:
<img width="1392" alt="Screen Shot 2022-06-20 at 09 44 35" src="https://user-images.githubusercontent.com/2046265/174550833-7ce22a27-da38-4072-a228-db5f53dd797d.png">

Turquoise on white seems like a very sub-optimal choice, at least doesn't work for my eyes.

**Comment by hoechenberger** [comment]
Yes, I like this idea

**Comment by Dev-satish** [comment]
@hoechenberger if this is an easy/good first issue shall I try for the same?

**Comment by marsipu** [comment]
@Dev-satish Thank you having a look into this.
But I think this event-color issue might better be resolved within mne itself, where the event-colors are set with [_make_event_color_dict](https://github.com/mne-tools/mne-python/blob/main/mne/viz/utils.py#L406)

**Comment by marsipu** [comment]
Would you agree @hoechenberger?

**Comment by hoechenberger** [comment]
Sounds like a plan!
